src/components/searchresult/common/PDFSearchResult.vue

```diff
@@ -60,6 +60,19 @@
         type: String,
         requred: true
       }
-    }
+    },
+    methods: {
+      checkForSnippetAvailability(query) {
+        if(query === '*.*' || query === '*:*' || query.includes('&pt=')) {
+          return false;
+        }
+        else {
+          return true;
+        }
```

Comment:
I actually stumbled upon a case where a ternary would have cleaner ;)))

Reply:
Yes! I'll do that!